src/data_utils/preprocessing_helper.py

```diff
@@ -0,0 +1,31 @@
+import json
+import os
+
+ABBREVIATIONS_FILES_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../..', 'Data', 'contractions')
```

Comment:
Is this contraction file dir or abbreviation dir path?

Reply:
It is done